ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -0,0 +1,258 @@
+## Arctica Greeter - Installation Guide for Ubuntu 16.04 LTS (Latest version)
+
+# Introduction
+
+The following document, documents how to install Ubuntu with Arctica Greeter. Make sure when typing the commands for installation, since Ubuntu differ between uppercase and lowercase letters because everything is case-sensitive. Everything that is highlighted with a black background is what you would type in the Terminal.
+
+## Stage one
+ 
+**Install Ubuntu:**
+When installing Ubuntu make sure to check both boxes, **Download Updates While Installing Ubuntu** and **Installation of 3rd party applications**, when they both are ticked then press continue. </span>
+![1](https://github.com/Korn20000/Testing/blob/master/img/1.png?raw=true)
+
+Next step choose **Erase Disk** and press Install Now.
+If it asks you **“Write the changes to disk?”**, ignore it and press continue.
+
+![2](https://github.com/Korn20000/Testing/blob/master/img/2.png?raw=true)

```

Comment:
Please replace absolute URLs by relative / local URLs. This comment stands for all further locations where href URLs are used.